### ISSUE TITLE
Revise Imports

### ISSUE THREAD

**Issue by mark-jensen** [OGMS/ogms#107]
Currently OGMS directly imports:
http://purl.obolibrary.org/obo/iao/dev/ontology-metadata.owl
http://protege.stanford.edu/plugins/owl/dc/protege-dc.owl
http://purl.obolibrary.org/obo/bfo.owl

As Sivaram [noted](https://groups.google.com/forum/#!topic/ogms-discuss/Vj8dTZOLpqA), OGMS currently has no object properties. The reason for this is because the above PURL for BFO redirects to the classes-only version of BFO, which is becoming the standard for many BFO users. But, it lacks object properties. The workaround is to import [RO-core](http://purl.obolibrary.org/obo/ro/core.owl). See [here](https://github.com/oborel/obo-relations/wiki/ROAndBFO). 

A problem is that can lead to duplication of some annotations, such a 'label' or 'definition'. This is a known issue with no simple fix, as @cmungall points out [here](https://github.com/oborel/obo-relations/issues/107). There exist some short-term workarounds to clean up, if we need to, but in the meantime I recommend OGMS import RO-core, as the importance of having object properties outweighs a few duplicate annotations.  

As a result we no longer need the DC and IAO metadata imported separately, since RO-core imports the [annotations module](http://purl.obolibrary.org/obo/ro/annotations.owl) from RO.

To sum up:
Delete:
http://purl.obolibrary.org/obo/iao/dev/ontology-metadata.owl
http://protege.stanford.edu/plugins/owl/dc/protege-dc.owl
Add:
http://purl.obolibrary.org/obo/ro/core.owl

I made [pull request](https://github.com/OGMS/ogms/pull/106) of my fork with these changes (also fixing a problem with IAO 'information content entity' & 'data item').
